src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin.csproj

```diff
@@ -40,8 +40,8 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.AzureStack.Management.Fabric.Admin, Version=0.2.0-preview">
-      <HintPath>..\..\..\packages\Microsoft.AzureStack.Management.Fabric.Admin.0.2.0-preview\lib\net452\Microsoft.AzureStack.Management.Fabric.Admin.dll</HintPath>
+    <Reference Include="Microsoft.AzureStack.Management.Fabric.Admin, Version=0.4.0-preview">
```

Comment:
What happened to 0.3.0-preview?

Reply:
I have no idea why the ps module doesn't ref 0.3.0-preview SDK, since we updated SDK version to 1.8.0-preview, I have change it to match the latest version.